app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert 30 == sum(split_integer(30, 4))
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(30, 5) == [6, 6, 6, 6, 6]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert [30] == split_integer(30, 1)
```

Comment:
```suggestion
    assert split_integer(30, 1) == [30]
```
Place the actual value first